src/rotation/rotations.controller.ts

```diff
@@ -87,10 +85,7 @@ export class RotationsController {
     @Query(ValidationPipe)
     findRotationQueryDto: FindRotationQueryDto,
   ): Promise<Partial<RotationEntity>[]> {
-    const {
-      month = getNextYearAndMonth().month,
-      year = getNextYearAndMonth().year,
-    } = findRotationQueryDto;
+    const { month = undefined, year = undefined } = findRotationQueryDto;
```

Comment:
```suggestion
    const { month, year } = findRotationQueryDto;
```
`findRotationQueryDto` 안에 `month`나 `year`가 없으면 undefined를 할당하지 않아도 undefined가 됩니다 ㅎㅎ